src/current/v23.2/physical-cluster-replication-overview.md

```diff
@@ -0,0 +1,132 @@
+- Avoid conflicts in data after recovery because the replication completes to a transactionally consistent state as of a certain point in time.
+
+## Features
+
+- **Asynchronous byte-level replication**: When you initiate a replication stream, it will replicate byte-for-byte all existing user data and associated metadata (e.g., zone configurations) in the primary cluster to the standby cluster asynchronously. From then on, it will continuously replicate user data and metadata in the primary to the standby. Physical cluster replication will automatically replicate changes related to operations such as, schema changes, user and privilege modifications, and zone configuration updates without any manual work.
+- **Transactional consistency**: When you cut over to the standby, you can specify `LATEST`, a certain point of time in the past, or future. The standby cluster will be in a transactionally consistent state as of a certain point in time after the cutover process. As a result, there is no need for conflict resolution.
+- **Maintained/improved RPO and RTO**: Depending on workload and deployment configuration, [replication lag]({% link {{ page.version.version }}/physical-cluster-replication-technical-overview.md %}) between the primary and standby is generally in the tens of seconds range. The cutover process from the primary cluster to the standby will take 5 minutes or under when completing a cutover to the latest replicated time using `LATEST`.
+- **Cutover to a timestamp in the past**: In the case of logical disasters or mistakes, you can cut over from the primary to the standby cluster to a timestamp in the past. This means that you can return the standby to a timestamp before the mistake was replicated to the standby. You can also configure the `WITH RETENTION` option to control how far in the past you can cut over to. {% comment %}link retention option here{% endcomment %}
+- **Monitoring**: To monitor the replication progress, you can use metrics available in the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}) and [Prometheus]({% link {{ page.version.version }}/monitor-cockroachdb-with-prometheus.md %}). For more detail, refer to [Physical Cluster Replication Monitoring]({% link {{ page.version.version }}/physical-cluster-replication-monitoring.md %}).
+
+{% comment %}- **Data verification on standby**: You can verify that the data on the standby cluster matches that on the primary by checking the fingerprint of the data on the primary to compare to the standby. We recommend running data verification checks regularly as part of your monitoring processes. Refer to the [Physical Cluster Replication Monitoring]({% link {{ page.version.version }}/physical-cluster-replication-monitoring.md %}) page for a guide and considerations on data verification.(not yet merged as of 10/10){% endcomment %}
```

Comment:
Data verification is merged now. But I recommend we document it in a standalone PR.

Reply:
Ah yes, I have a docs issue around this, as soon as I know which 23.2 testing release it is in, I will start to document and add to the monitoring page for PCR.